package.json

```diff
@@ -3,7 +3,7 @@
   "name": "terraform",
   "displayName": "HashiCorp Terraform",
   "description": "Syntax highlighting and autocompletion for Terraform",
-  "version": "2.33.2024090609",
+  "version": "2.33.0",
```

Comment:
I'm wondering whether we need to change this to 2.34.0 or release a new pre-release immediately after for pre-release folks to get the latest changes 🤔

Reply:
We release a new pre-release immediately after